jgiven-core/src/test/java/com/tngtech/jgiven/impl/JGivenAppenderImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.tngtech.jgiven.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+
+@Plugin(name = "JGivenAppender", category="Core", elementType="appender", printObject=true)
```

Comment:
maybe rather in a testutil folder? 
Als I wander whether one could have used the appender manually.